src/I18n/Date.php

```diff
@@ -45,6 +45,22 @@ class Date extends MutableDate implements I18nDateTimeInterface
      */
     protected static $_toStringFormat = [IntlDateFormatter::SHORT, -1];
 
+    /**
+     * The format to use when when converting this object to json
```

Comment:
JSON is probably an acronym here.

Reply:
I simply cut - pasted this from the `DateTimeTrait`. There's a double "when" too in the statement :)

Reply:
`... when when ...`